### DISCUSSION TITLE
HA (HASSOS) + Apache reverse proxy

### DISCUSSION THREAD

**Discussion by barbatrukko** [indomus/forum#1710] [HUB: Home Assistant]
Salve, fino a qualche tempo fa utilizzavo HA con i plugin duckDNSe LEncrypt per accedere ad HA dall'esterno.
Facevo il PortForwarding delle port a443 esterna del router sulla 8123 del raspPi che esegue HA.
Poichè ho aggiunto altri servizi che hanno bisogno a loro volta della porta 443, ho di recente installato un apache che fa da reverse proxy riceve tutte le chiamate sulla porta 443 smistandole ai vari servizi.
L'unico servizio che non riesco a configurare è HA.
Ho anche disattivato https su HA in modo che l'apache accede in chiaro internamente e lo trasforma in https esternamente... ma nulla
ottengo un bad request 400
Io localmente accedo tranquillamente sulla porta 8213 senza https

nel file di configurazione di HA ho anche aggiunto
```
http:
  use_x_forwarded_for: true
  trusted_proxies:
    - 192.168.0.XX
```

Qualcuno di voi ha un accrocco simile?

**Comment by snakuzzo** [comment]
la disabilitazione non è che andava provata...ma andava proprio fatta. Non aveva alcun senso lasciare HA in https.
Probabilmente ottieni HTTP 400 perché l'indirizzo chiamante non è tra i trusted proxy....e nel tuo caso dovrebbe essere quello di apache. Ad ogni modo l'ip da aggiungere lo puoi vedere nei log di HA

**Reply by barbatrukko** [reply]
Io ho aggiunto l'ip del proxy, ma il risultato non cambia....
in https non era utile (il doppio criptaggio) ma doveva funzionare comunque (come funziona su altri servizi)
Tuttavia ora è in http e non funziona lo stesso.
Quindi ora che altro devo provare?

**Reply by snakuzzo** [reply]
Guarda il log di HA...te l'ho scritto prima. L'hai fatto ?
